lib/plugins/aws/invokeLocal/index.js

```diff
@@ -272,19 +266,20 @@ class AwsInvokeLocal {
     return this.invokeLocalDocker();
   }
 
-  checkDockerDaemonStatus() {
-    return spawnExt('docker', ['version']).catch(() => {
+  async checkDockerDaemonStatus() {
+    try {
+      return spawnExt('docker', ['version']);
```

Comment:
It misses `await`

Reply:
When returning a promise, `await` is not needed. But I'll add it anyway.

Reply:
In `try` clause it is needed, otherwise rejection won't be caught

Reply:
Right! 🤦‍♂️